types/postcss-inline-svg/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "es2018.promise"
```

Comment:
is this because of?
`Property 'finally' does not exist on type 'Promise<Result>'.`

Reply:
Yes. But also (according to [postcss's tsconfig](https://github.com/postcss/postcss/blob/f8b678ca1e867acdcd10d0efc5c2a8f575dfc742/tsconfig.json#L3) from 8.1.4) is it worth using full es2018 lib instead of es6 + promise from es2018?